docs/Tutorials/Configuration.md

```diff
@@ -68,17 +68,18 @@ A "path" like `Server.Ssl.Protocols` looks like the below in the file:
 }
 ```
 
-| Path | Description | Docs |
-| ---- | ----------- | ---- |
-| Server.Ssl.Protocols | Indicates the SSL Protocols that should be used | [link](../Certificates) |
-| Server.Request | Defines request timeout and maximum body size | [link](../RequestLimits) |
-| Server.AutoImport | Defines the AutoImport scoping rules for Modules, SnapIns and Functions | [link](../Scoping) |
-| Server.Logging | Defines extra configuration for Logging, like masking sensitive data | [link](../Logging/Overview) |
-| Server.Root | Overrides root path of the server | [link](../Misc/ServerRoot) |
-| Server.Restart | Defines configuration for automatically restarting the server | [link](../Restarting/Types/AutoRestarting) |
-| Server.FileMonitor | Defines configuration for restarting the server based on file updates | [link](../Restarting/Types/FileMonitoring) |
-| Web.TransferEncoding | Sets the Request TransferEncoding | [link](../Compression/Requests) |
-| Web.Compression | Sets any compression to use on the Response | [link](../Compression/Responses) |
-| Web.ContentType | Define expected Content Types for certain Routes | [link](../Routes/Utilities/ContentTypes) |
-| Web.ErrorPages | Defines configuration for custom error pages | [link](../Routes/Utilities/ErrorPages) |
-| Web.Static | Defines configuration for static content, such as caching | [link](../Routes/Utilities/StaticContent) |
+| Path                              | Description                                                             | Docs                                       |
+| --------------------------------- | ----------------------------------------------------------------------- | ------------------------------------------ |
+| Server.Ssl.Protocols              | Indicates the SSL Protocols that should be used                         | [link](../Certificates)                    |
+| Server.Request                    | Defines request timeout and maximum body size                           | [link](../RequestLimits)                   |
+| Server.AutoImport                 | Defines the AutoImport scoping rules for Modules, SnapIns and Functions | [link](../Scoping)                         |
+| Server.Logging                    | Defines extra configuration for Logging, like masking sensitive data    | [link](../Logging/Overview)                |
+| Server.Root                       | Overrides root path of the server                                       | [link](../Misc/ServerRoot)                 |
+| Server.Restart                    | Defines configuration for automatically restarting the server           | [link](../Restarting/Types/AutoRestarting) |
+| Server.FileMonitor                | Defines configuration for restarting the server based on file updates   | [link](../Restarting/Types/FileMonitoring) |
+| Server.RouteOrderMainBeforeStatic | Changes the way routes are processed.                                   | [link](../Routes/Utilities/StaticContent)  |
```

Comment:
done

Reply:
In the `Configuration.md` file, `Server.RouteOrderMainBeforeStatic` was updated to be `Server.Web.Static.ValidateLast` instead of just `Web.Static.ValidateLast`.

The change to `Set-PodeWebConfuguration` is good.

Reply:
I see I made the wrong assumption on how Web configuration works